src/settings/types.rs

```diff
@@ -163,6 +164,12 @@ impl Default for SerializationFormat {
                 return Self::Github;
             }
         }
+
+        if let Ok(gitlab_ci) = env::var("GITLAB_CI") {
```

Comment:
This may not run anymore because default value is not used anymore. 
`format: options.format.unwrap_or_default()` was remove from `src/settings/configuration.rs` on https://github.com/charliermarsh/ruff/pull/1219/

Reply:
Ahh yeah, that actually seems like a bug to me. The unwrap step got moved to `src/settings/mod.rs` (`format: config.format.unwrap_or(SerializationFormat::Text)`), but should likely still use `default`. If you want to change that here, that'd be welcome; or as a separate PR, or I can handle it later.

Reply:
Sure, I can add it here as they are closely related. :)